hw03_cruzmendeze.py

```diff
@@ -0,0 +1,141 @@
+#####################################################################################
+#
+# Author: Eric Cruz-Mendez
+# Username: cruzmendeze
+#
+# Assignment: HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Google Doc Link: https://docs.google.com/document/d/1P5qvQ1eb9AhFPyGV7J0JwgZ8XdeqU9qpbF82LSsu6yM/edit?usp=sharing
+#
+#####################################################################################
+
+import turtle
+turtle.colormode(255)  # Set colormode to RGB value
+
+# Function that makes the roof
+def roof(shape):
+    shape.color(255, 182, 193)  # RGB values for unnamed color
+    shape.begin_fill()
+    for side in range(3):       # Loop for triangular roof
+        shape.forward(300)
+        shape.left(120)
+    shape.end_fill()  # Tells Python the shape is complete; now fill it
+    shape.penup()
+
+# Function that makes the main house rectangle
+def house(shape):
+    shape.color('pink')
+    shape.pendown()
+    for side in range(2):   # Makes the square frame for the house
+        shape.forward(300)
+        shape.right(90)
+        shape.forward(250)
+        shape.right(90)
+    shape.end_fill()
+
+# Function that make windows
+def window(shape):
+    shape.penup()
+    shape.color('pink')
+    for side in range(2):
+        shape.forward(50)
+        shape.right(90)
+    for side in range(2):
+        shape.pendown()
+        shape.forward(-40)
+        shape.right(90)
+        shape.forward(-40)
+        shape.right(90)
+    shape.penup()
+    shape.forward(-200)
+    shape.right(90)
+    shape.forward(-40)
+    shape.pendown()
+    for side in range(2):
+        shape.forward(40)
+        shape.right(90)
+        shape.forward(-40)
+        shape.right(90)
+    shape.penup()
+# Function that makes door
+def door(shape):
```

Comment:
Pay attention to the gray squigglies, they indicate style guidelines. This one says all function definitions should have two blank lines before them. 

Fix this in multiple places.

Reply:
I have added the blank lines.